docs/guides/trivy.md

```diff
@@ -0,0 +1,47 @@
+# Moving towards configuration scanning with Trivy
+Overtime we've taken [trivy][trivy] to be the go-to scanning tool for a vareity of things. This also includes terraform scanning.
+
+This section describes some differences between Trivy and tfsec.
+
+| Feature              | Trivy                                                  | tfsec                |
+|----------------------|--------------------------------------------------------|----------------------|
+| Policy Distribution | Embedded and Updated via Registry                      | Embedded             |
+| Custom Policies      | Rego                                                   | Rego, JSON, and YAML |
+| Supported Formats    | Dockerfile, JSON, YAML, Terraform, CloudFormation etc. | Terraform  Only      |
```

Comment:
wrong way?

Reply:
wrong way in what sense? you want the tfsec column first?